packages/block-library/src/query/edit/inspector-controls/enhanced-pagination-control.js

```diff
@@ -25,19 +25,19 @@ export default function EnhancedPaginationControl( {
 		);
 	} else if ( enhancedPagination ) {
 		help = __(
-			"Browsing between pages won't require a full page reload, unless non-compatible blocks are detected."
+			'Reload the full page—instead of just the posts list—when visitors navigate between pages.'
 		);
 	} else if ( hasUnsupportedBlocks ) {
 		help = __(
-			"Force page reload can't be disabled because there are non-compatible blocks inside the Query block."
+			"Reload full page can't be disabled because there are non-compatible blocks inside the Query block."
```

Comment:
One important note. When there are non-compatible blocks, then the full page navigation, as in older WordPress versions, needs to happen. Eventually, we should aim to remove this compatibility check and instead handle everything on the front end using a progressive enhancement approach. For example, when the partial page change withouth full reload is enabled, we try to go to the next page and if Interactivity API detects something went wrong, it should fall back to the usual behavior when someone click a link.